nat-lab/tests/telio.py

```diff
@@ -522,6 +522,9 @@ def __init__(
     async def run(
         self, meshmap: Optional[Meshmap] = None, telio_v3: bool = False
     ) -> AsyncIterator["Client"]:
+        if isinstance(self._connection, DockerConnection):
```

Comment:
During `test_event_content_meshnet_node_upgrade_direct` where beta node is deleted and started again, only 2nd instance log will be saved and 1st log file will be overwritten, If I understand correctly.

Either create file iterations or maybe attach tcpdump to connection instead of telio instance? But not sure about that

Reply:
Thanks, that's a very good point and I totally missed that! I went with counter in the last part of the file name.